README.md

```diff
@@ -56,6 +60,18 @@ Tool                  | Description
 [Kaminari]            | Elegant pagination for any sort of collection
 [Ransack]             | Provides a simple search API to query your data
 
+## Acknowledgements
+
+Thanks to [Greg] for creating and opensourcing this project.
+
+Thanks to all the people that ever contributed to this project through [code] or
+other means such as bug reports, issue triaging, feature suggestions, code
+snippet tips, slack dicussions and so on.
+
+Thanks to [Tidelift][tidelift] and our Tidelift subscriptors.
```

Comment:
```suggestion
Thanks to [Tidelift][tidelift] and all our Tidelift subscribers.
```

Reply:
Subscriptors?? Sometimes I'm embarrased by my English :rofl:

Reply:
Haha sorry, don't feel bad. 😂 You do great all around! I figure with this you are learning too. Thanks! 👏🏻